KarhooUISDK/Translations/Base.lproj/Localizable.strings

```diff
@@ -89,6 +89,12 @@
 
 "kh_uisdk_acc_ride_details" = "Go to ride details";
 
+"kh_uisdk_accessibility_icon_cross" = "Cross icon";
+
+"kh_uisdk_accessibility_label_close_screen" = "Close the screen";
```

Comment:
Looks like we already have `kh_uisdk_acc_close_button` and `kh_uisdk_close_the_screen`. Please make sure we use only one version of that value (so we can remove not needed keys) and add proper information to Android ticket